README.md

```diff
@@ -27,7 +27,7 @@ http://localhost:8080 will have the rchab api in the vm and on your host.
 `flyctl` can be configured to use a locally running version of rchab with:
 
 ```shell
-FLY_REMOTE_BUILDER_HOST_WG=1 FLY_RCHAB_OVERRIDE_HOST=tcp://127.0.0.1:2375 LOG_LEVEL=debug fly deploy --remote-only
+FLY_REMOTE_BUILDER_HOST_WG=1 FLY_RCHAB_OVERRIDE_HOST=tcp://127.0.0.1:2376 LOG_LEVEL=debug fly deploy --remote-only
```

Comment:
For local testing, you'd probably still want to connect to 2375, right?

Reply:
yup